### PR TITLE
[Training] update templates to use new, more accessible box style

### PR DESCRIPTION
**Pull request by shiltemann**
Goes with https://github.com/galaxyproject/training-material/pull/3633

ping @hexylena

### CODE REVIEW

**Review by hexylena — APPROVED** [review]
Looks fantastic! And nice that it can simplify some of the ugly templating bits.

File: planemo/training/tutorial.py [hunk 1]
```diff
@@ -56,15 +56,15 @@
...
+> <hands-on-title> Hands-on: Task description </hands-on-title>
 >
 > 1. {{ '{%' }} tool [{{tool_name}}]({{tool_id}}) {{ '%}' }} with the following parameters:{{inputlist}}{{paramlist}}
 >
 >    ***TODO***: *Check parameter descriptions*
 >
 >    ***TODO***: *Consider adding a comment or tip box*
 >
->    > ### {{ '{%' }} icon comment {{ '%}' }} Comment
+>    > <comment-title> Comment </comment-title>
```
Comment (hexylena) [inline planemo/training/tutorial.py:67]: This can be empty, now it will read `Comment: Comment`

Comment (hexylena) [reply planemo/training/tutorial.py:67]:
```suggestion
>    > <comment-title></comment-title>
```

File: planemo/training/tutorial.py [hunk 2]
```diff
@@ -195,7 +194,7 @@
 
 ## Get data
 
-> ### {{ '{%' }} icon hands_on {{ '%}' }} Hands-on: Data upload
+> <hands-on-title> Hands-on: Data Upload </hands-on-title>
```
Comment (hexylena) [inline planemo/training/tutorial.py:197]:
```suggestion
> <hands-on-title> Data Upload </hands-on-title>
```

**Comment by mvdbeek** [comment]
Thanks!

**Pull request merged**